style.css

```diff
@@ -0,0 +1,18 @@
+body{
+background: linear-gradient(45deg, moccasin, darksalmon);
```

Comment:
Two spaces are used for indents inside the rules.
Before the opening brace ( { ) put one space.
These recommendations are indicated in the guideline, which had to be followed http://codeguide.academy/html-css.html